scripts/sftpCommands.sh

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+
+VERSION=$1
+
+# When sftp commands are run in batch mode (-b flag), commands prefixed with "-" do not fail the
+# entire script. By prefixing mkdir with "-", this will not fail if the directory already exists.
+echo "-mkdir $VERSION"
+echo "cd $VERSION"
+echo "put ./dist/alloy.js"
+echo "put ./dist/alloy.min.js"
+echo "bye"
```

Comment:
Can we verify that it uploaded or it's overkill?

Reply:
Good idea. I've added some commands to this script to check that the files are available on the CDN.